src/components/PDFView/PDFPasswordForm.js

```diff
@@ -67,7 +69,15 @@ function PDFPasswordForm({isFocused, isPasswordInvalid, shouldShowLoadingIndicat
         if (!textInputRef.current) {
             return;
         }
-        textInputRef.current.focus();
+        focusTimeoutRef.current = setTimeout(() => {
```

Comment:
I would add the comment with the link to Slack comment posted by Rory explaining why we should use `setTimeout` here.

Reply:
I think we do - in general we are trying to avoid using `setTimeout` in the app so describing why we had to use it is a good habit.